shell/src/main/java/tachyon/shell/TFsShell.java

```diff
@@ -113,6 +113,53 @@ public int cat(TachyonURI path) throws IOException {
   }
 
   /**
+   * Copies a file or directory specified by argv from the under filesystem to the filesystem. 
```

Comment:
If I have `/file` in under filesystem but not in Tachyon now, the call to `load` would not succeed? Need to call `loadufs` first? Maybe better to clarify the difference from `loadufs`.